check-branch-name/action.yml

```diff
@@ -0,0 +1,22 @@
+name: "Check if the branch name follows our naming convention."
+description: "Action that checks if the name of the branch is following the our branch naming convention."
+
+runs:
+  using: composite
+  steps:
+    - uses: jungwinter/split@master
```

Comment:
Instead of using a third party action for splitting a string, let us go with the `cut` command. For using this command, you specify the "delimiter" and the "field" you want to retain after cutting the string.

```yml
- name: "Get the prefix of the branch"
  if: github.ref_type == 'branch'
  shell: bash
  run: |
    branch_prefix=$(cut -d '/' -f 1)
    echo "BRANCH_PREFIX=$branch_prefix" >> $GITHUB_ENV
```